keras_hub/src/layers/modeling/non_max_supression.py

```diff
@@ -3,21 +3,18 @@
 import keras
 from keras import ops
 
-# TODO: https://github.com/keras-team/keras-hub/issues/1965
-from keras_hub.src.bounding_box import converters
-from keras_hub.src.bounding_box import utils
-from keras_hub.src.bounding_box import validate_format
+from keras_hub.src.api_export import keras_hub_export
 
 EPSILON = 1e-8
 
 
+@keras_hub_export("keras_hub.layers.NonMaxSupression")
 class NonMaxSuppression(keras.layers.Layer):
     """A Keras layer that decodes predictions of an object detection model.
 
     Args:
         bounding_box_format: The format of bounding boxes of input dataset.
-            Refer
-            TODO: link keras core bounding box docs
+            Refer:
```

Comment:
Just plain text here. I don't think Keras.io could render some sort of block inside a arg. E.g. "Refer to blah blah for more details on supported bounding box formats."

Also cut this if we don't have an actual content link

Reply:
Done